tests/page-objects/reports/reports.wdio.page.js

```diff
@@ -110,9 +115,77 @@ const reportsListDetails = async () => {
   return reportDetails;
 };
 
+const collapseSelection = async () => {
+  await (await itemSummary()).click();
+  expect(await (await reportBodyDetails()).isExisting()).to.be.false;
+};
+
+const deleteSelectedReports = async (savedUuids) => {
+  const deleteAllButton = () => $('.action-container .detail-actions .delete-all');
+  const confirmButton = () => $('.btn.submit.btn-danger');
+
+  await (await deleteAllButton()).click();
+  await (await confirmButton()).click();
+  await (await firstReport ()).waitForDisplayed();
+  // make sure the reports are deleted
+  expect(await reportsByUUID(savedUuids[1]).length).to.equal(1);
+};
+
+const deselectAll = async () => {
+  const deselectAllButton = await $('.action-container .deselect-all');
+  await deselectAllButton.click();
+  const count = await $('#reports-content .selection-count > span');
+  expect(await count.isExisting()).to.be.false;
+  expect(await $$(reportBody).length).to.equal(0);
+};
+
+const expandSelection = async () => {
+  await (await itemSummary()).click();
+  await (await $(reportBodyDetailsSelector)).waitForDisplayed();
+};
+const selectAll = async () => {
+  await (await $('.action-container .select-all')).click();
+  await (await $('#reports-content .selection-count > span')).waitForDisplayed();
+  expect(await $$(reportBody).length).to.equal(3);
+};
+
+const selectSeveralReports = async (savedUuids) => {
+  const firstCheck = (await reportsByUUID(savedUuids[0]))[0].$(checkCss);
+  await firstCheck.click();
+  const secondCheck = (await reportsByUUID(savedUuids[2]))[0].$(checkCss);
+  secondCheck.click();
+  await browser.pause(1000);
+  expect(await (await $$(reportBody)).length).to.equal(2);
+};
+
+const selectReport = async (savedUuids) => {
```

Comment:
I think this should be changed to only receive one uuid, the one that it should select, instead of receiving an array. 
It's a very weird interface, to pass an array of uuids and only select the first one.